src/host/cookedRead.hpp

```diff
@@ -112,7 +112,7 @@ class CookedRead final : public ReadData
     COORD _promptStartLocation;
     // the location of the cursor before a popup is launched
     COORD _beforePopupCursorPosition;
-    CommandHistory* _pCommandHistory; // non-ownership pointer
+    std::shared_ptr<CommandHistory> _pCommandHistory; // shared pointer

```

Comment:
nit: the comment on this line is redundant now.

Reply:
ok...